mysite/templates/mysite/home.html

```diff
@@ -0,0 +1,18 @@
+{% extends "mysite/base_generic.html" %}
+
+{% block content %}
+
+{% if user.is_authenticated %}
+    <h2>Welcome {{ user.username }}</h2>
+    {% if user.user_role == 'recruiter' %}
```

Comment:
user role should be defined in some Model OR Form as variables and related choices filed should use that variables. That roles variables should be pass into templates to apply any related checks. This is to prevent duplication.

Reply:
Feedback Incorporated.